mycpp/examples/test_func_method_name_conflict.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python2
+"""
+test_func_method_name_conflict.py
+"""
+
+import os
+
+
+def a():
+    # type: () -> int
+    return 1
+
+
+class A:
+
+    def __init__(self):
+        # type: () -> None
+        pass
+
```

Comment:
To test the previous bug, we could add another method function that calls a() before the method is defined

And maybe a free function that calls the free function a(), etc.